packages/wdio-testingbot-service/src/service.ts

```diff
@@ -151,13 +160,14 @@ export default class TestingBotService {
             return this.updateJob(global.browser.sessionId, failures)
         }
 
+        let browser = global.browser
```

Comment:
@martinfrancois This is needed so Typescript will detect type of multiremote browser object correctly
```
        if (!global.browser.isMultiremote) {
            log.info(`Update job with sessionId ${global.browser.sessionId}, ${status}`)
            return this.updateJob(global.browser.sessionId, failures)
        }

        let browser = global.browser
```